webapp/components/sidebar_header.jsx

```diff
@@ -5,40 +5,66 @@ import React from 'react';
 
 import Client from 'client/web_client.jsx';
 import PreferenceStore from 'stores/preference_store.jsx';
+import UserStore from 'stores/user_store.jsx';
 import * as Utils from 'utils/utils.jsx';
 
 import SidebarHeaderDropdown from './sidebar_header_dropdown.jsx';
 import {Tooltip, OverlayTrigger} from 'react-bootstrap';
 
 import {Preferences, TutorialSteps, Constants} from 'utils/constants.jsx';
 import {createMenuTip} from 'components/tutorial/tutorial_tip.jsx';
+import StatusDropdown from './status_dropdown.jsx';
 
 export default class SidebarHeader extends React.Component {
     constructor(props) {
         super(props);
 
         this.toggleDropdown = this.toggleDropdown.bind(this);
         this.onPreferenceChange = this.onPreferenceChange.bind(this);
+        this.onStatusChange = this.onStatusChange.bind(this);
```

Comment:
Thanks @hmhealey, I thought that the explicit binding style was preferred since it is widespread in the codebase. I'll update my changes accordingly